### ISSUE TITLE
Change wording for "At a glance" dashboard widget

### ISSUE THREAD

**Issue by Zodiac1978** [pluginkollektiv/antispam-bee#171]
> Perhaps the wording of that text should be adjusted in the “at a glance” box on the dashboard to reflect the actual status. Something like “{number} spam comments detected and deleted automatically” or something.
> The way it is now, it’s misleading and confusing.

https://wordpress.org/support/topic/list-of-blocked-comments-2/

This is a problem because the meaning of this number is switching according to the delete option:
https://github.com/pluginkollektiv/antispam-bee/wiki/en-Documentation#spam-counter-on-the-dashboard

I think the draft is much to long, but maybe there is a way to better reflect the change.

**Comment by websupporter** [comment]
When the option to display the number in the Glance-Box is set, every as spam detected comment increments a counter. Basically I see this number more as a gimmick "this is how much comments ASB has already eaten". But the link indicates there would be some kind of functionality behind this number. As the user in the support forum points out: The link does not lead to a list, which is reasonable to assume.

If we change the wording the way it is suggested, this might be irritating though with the current implementation. Lets say at counter=100 the user decides to switch the "do-delete"-option and it is not deleted anymore At counter=200 it would either say "200 spam comments have been detected and deleted", which would be wrong or "200 spam comments have been detected" and once the user jumps into the "spam"-list, he only sees 100 and wonders as well.

To link to the spam-list (instead of the options as it is the current state) would also provoke the problem why are the numbers different.

Basically I think, we should remove the link, so we do not imply, behind this number would be any kind of functionality besides "look, so many spam has been eaten".

**Comment by websupporter** [comment]
The code in question for removing the link: https://github.com/pluginkollektiv/antispam-bee/blob/master/antispam_bee.php#L700-L710

**Comment by websupporter** [comment]
Once, we have changed this, we should also (as discussed in Slack) change the misleading Wikitext https://github.com/pluginkollektiv/antispam-bee/wiki/de-Dokumentation#spam-anzahl-auf-dem-dashboard-anzeigen
